docs/marine-book/marine-runtime/configuration-file.md

```diff
@@ -71,19 +72,19 @@ where `?` represents an optional field and `specificator` is one from the follow
 * `P`, `Pb` - petabyte
 * `Pi`, `PiB` - pebibyte
 
-Note, that practically the current limit of Wasm memory is limited to 4 GiB, so you shouldn't use specifiers bigger than gibibyte. Additionally, all specificators are case-insensitive.
+Note, that practically the current limit of Wasm memory is limited to 4 GiB per module, so a service cannot consume more than `modules_number * 4 GiB`. Additionally, all specificators are case-insensitive.
```

Comment:
Don't really know how to address this correctly, but there can be a pottential confusion from the user with the fact that for worker max is 2 GB of memmory (1 Compute Unit). So it doesn't really matter for the fluence user that module can have 4 GiB, it matters only for Marine user

Reply:
but I understand that this is a marine doc anyway

Reply:
> worker max is 2 GB of memmory

don't think that's correct at the unit level; also, there may be more than one CU bought by the developer to cover their execution requirements. see FLIP 26.

Reply:
don't understand what you mean by saying "at the unit level"
can't easily find something about this in FLIP 26, could you please help. Also is buying more then 1 CU something planed for MVM?

Reply:
That is the Marine documentation, it should describe Marine capabilities and limitations, not Nox/worker/Fluence Protocol ones. So i thing that it is wrong to reference workers/CUss